spec/controllers/addresses_controller_spec.rb

```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+RSpec.describe AddressesController, type: :controller do
+
+end
```

Comment:
This file is empty. Is it not needed? Do we plan on doing controller testing in addition to API "acceptance" testing?